src/main/java/com/hartwig/platinum/config/PlatinumConfiguration.java

```diff
@@ -50,7 +50,7 @@ default boolean inCluster() {
 
     Optional<String> cluster();
 
-    Optional<String> apiUrl();
+    Optional<HmfConfiguration> hmfConfiguration();
 
```

Comment:
Also a quick note on config, the term `sampleIds` is used here in the config when the values are used as either biopsies or tumor sample names. This choice was made to make it easier for the users (ie AUS) who call them sample ids and don't really know/care about whether something is technically a reference to a biopsy or sample in different contexts